angular/src/components/add-edit.component.ts

```diff
@@ -150,9 +151,9 @@ export class AddEditComponent implements OnInit {
     }
 
     async init() {
-        const myEmail = await this.userService.getEmail();
+        const myEmail = this.accountService.activeAccount?.email;
```

Comment:
Yes, active account should always be set unless logged in. We do check it sometimes when it could be null, so I was doing null propagation pretty much everywhere. I think making activeAccount private and supplying helpers is safer, and will swap to that.